pkgs/ffigen/lib/src/strings.dart

```diff
@@ -170,6 +170,7 @@ const formatVersion = 'format_version';
 const symbolFileFormatVersion = '1.0.0';
 const files = 'files';
 const usedConfig = 'used-config';
+const dartName = 'dartName';
```

Comment:
What about `'dart-name'` instead? That way the symbols.yaml file would be more consistent with the config.yaml

Reply:
Ah right, done.